Makefile

```diff
@@ -97,16 +97,16 @@ endif
 manifests: yq controller-gen kustomize ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role webhook paths="./..." crd output:crd:artifacts:config=config/crd/bases
 	$(CONTROLLER_GEN) rbac:roleName=manager-role webhook paths="./..." crd output:crd:artifacts:config=deploy/helm/grafana-operator/crds
-	yq -i '(select(.kind == "Deployment") | .spec.template.spec.containers[0].env[] | select (.name == "RELATED_IMAGE_GRAFANA")).value="$(GRAFANA_IMAGE):$(GRAFANA_VERSION)"' config/manager/manager.yaml
+	$(YQ) -i '(select(.kind == "Deployment") | .spec.template.spec.containers[0].env[] | select (.name == "RELATED_IMAGE_GRAFANA")).value="$(GRAFANA_IMAGE):$(GRAFANA_VERSION)"' config/manager/manager.yaml
```

Comment:
is the idea to allow different binaries to be provided from env vars? If so, what would you replace `yq` with? So far we always had the assumption that `yq` needs to be present on the system.

Reply:
Partially, only `PATH` is checked, but according to the `YQ:` target you never assumed that `yq` was present, at least in the Makefile.
But if that's the official stance on `yq`, I could remove the `YQ:` target instead?